express-zod-api/src/routing.ts

```diff
@@ -23,18 +24,32 @@ export interface Routing {
 
 export type Parsers = Partial<Record<ContentType, RequestHandler[]>>;
 
+const lineUp = (methods: Array<Method | AuxMethod>) =>
+  methods // options is last:
+    .toSorted((a, b) => +(a === "options") - +(b === "options"))
```

Comment:
perhaps we're fine to `.sort()` in place

```suggestion
    .sort((a, b) => +(a === "options") - +(b === "options"))
```